.gitignore

```diff
@@ -15,6 +15,9 @@ torch_xla/csrc/XLANativeFunctions.h
 torch_xla/csrc/RegisterXLA.cpp
 torch_xla/csrc/RegisterAutogradXLA.cpp
 
+# Directory autogenerated by full_codegen
+torch_xla/csrc/generated/
```

Comment:
I noticed the new generated directory getting tracked by git, so I've added to the ignore list. Let me know if this is okay.

Reply:
Thanks!